swiknaba/kirei
# Replace Sinatra with a PORO Router based on Rack, adds a test app

how to test:

```shell
cd spec/test_app

bundle exec rake db:create
bundle exec rake db:migrate

bundle exec puma
```

create some test records in the `airports` table, then head to localhost `http://0.0.0.0:9292/airports` and verify airports are rendered.
